config/locales/en.yml

```diff
@@ -85,6 +85,7 @@ en:
         title: Region
       sharepoint_link:
         title: SharePoint link
+        value: Open the project folder on SharePoint in a new tab
```

Comment:
This is somewhere else where I think we should be explicit, my understanding is that this is the establishment SharePoint folder.